app/posts/manage-your-screening/2025/04/2025-04-28-insights-from-team-invite-survey-april-2025.md

```diff
@@ -0,0 +1,120 @@
+
+
+### 2. Viewing ‘normal results in app:
+
+90% of respondents stated they would be willing to view their normal results in the NHS app.
+
+![A Three-part bar chart comparing patient comfort levels for different communication methods of breast screening results: NHS App, SMS, and letter. 160 people comfortable with the NHS App, 44 feel neutral and 22 uncomfortable. 132 are comfortable with SMS, 39 are neutral, and 52 feel uncomfortable. 208 participants are comfortable receiving results via letter, 16 feel neutral, and 8 uncomfortable](normal-results-survey-graph2.png)
+
+#### Impact on Mays: 
+
+A key question for Mays is how appealing is the idea of viewing results in the app to users and more importantly what the reasoning for not wanting to view results in app are.  
+
+We can then look for design decisions that act to increase users’ level of comfort with viewing normal results in app. 
+
+Whilst the group polled skew towards NHS app users and those that are digitally savvy it is useful to know that a high percentage of such users are comfortable with the idea of viewing normal results.  Meaning there is an  immediate useful and beneficial outcome for a good proportion of the NHS app user base. 
+
+#### User needs this finding relates to: 
+
+* CN011 - As a participant, I need a choice of communication channels, so that I can receive correspondence in accordance with my preferences and needs 
+
+* CN002 - As a participant, I need all information around the screening service to be readily available in a format that suits me, so that my needs can be met to help me make an informed decision whether to attend screening or not 
+
+* CN011 - As a participant, I need a choice of communication channels, so that I can receive correspondence in accordance with my preferences and needs 
+
+## Ranking importance of results information 
+
+Participants were asked to rank the information from most to least important for what they think should be within communications about Breast Screening normal results: 
+
+1.The result  
+
+2.What to do if you notice any breast cancer symptoms  
+
+3.How to ask questions about the result  
+
+4.A link to further guidance  
+
+5.Your breast screening office phone number  
+
+6.Your NHS Number  
+
+7.When you will next be invited  
+
+8.You will stop being invited to appointments after your 71st birthday  
+
+9.How to request this information in an alternative format  
+
+10.You can still self-refer for appointments after your 71st birthday  
+
+11.A thank you for your attendance  
+
+12.Disclaimer that the result doesn't mean you will never have cancer  
+
+13.The date and location of the screening appointment you attended  
+
+14.A reminder to update your contact details with your GP  
+
+15.A sign off at the end of the message such as 'kind regards' 
```

Comment:
These don't get recognised as a numbered list correctly. I think you need a space after the number, and no linebreak between them.